src/transformers/models/qwen2_vl/image_processing_qwen2_vl.py

```diff
@@ -20,9 +20,11 @@
 """Image processor class for Qwen2-VL."""
 
 import math
+from functools import lru_cache
 from typing import Dict, List, Optional, Union
 
 import numpy as np
+from numba import jit, prange
```

Comment:
let's make this a soft dep! 
(Meaning we protect the import with is numba available! 🤗 
And we need a bit of doc to say we recommend using numba. 
Users might wonder why prioritize this over torch with the Fast implementation so doc is needed! 🤗